templates/benchmark/pallet-weight-template.hbs

```diff
@@ -1,8 +1,25 @@
-{{header}}
```

Comment:
Let's only change the weight format and leave the rest changed: header, clippy hints, `LitentryWeight` name...